core/main.go

```diff
@@ -135,11 +174,21 @@ func main() {
 		edgex-vault-worker | ERROR: 2018/10/20 10:52:55 Import Policy HTTP Status: 500 Internal Server Error (StatusCode: 500)
 		edgex-vault-worker | ERROR: 2018/10/20 10:52:55 Fatal Error importing Admin policy in Vault.
 	*/
-	time.Sleep(5 * time.Second)
+	for {
```

Comment:
there should be some kind of timeout or attempt limit in this for loop, otherwise it may get into a situation where it loops forever waiting for vault to be ready when it never will be